playbook/app/pb_kits/playbook/pb_loading_inline/_loading_inline.tsx

```diff
@@ -0,0 +1,57 @@
+/* @flow */
```

Comment:
Once it's converted to typescript, you can remove the flow at the top @carloslimasd